schemas/rum/error-schema.json

```diff
@@ -131,6 +131,11 @@
                 }
               },
               "readOnly": true
+            },
+            "bundle_version": {
```

Comment:
I think we need to use version for this. 
A lot of datadog systems use version to detect regressions and such. Introducing a new version "facet" could be problematic.

Reply:
This is a very good point, I'm happy to discuss it.

## A bit more context
First let me give more context to make sure we're on the same page: every OTA update targets a specific native version ("version" in the schema). So they usually reset every time it targets a new native version.
i.e. the versions of the javascript running in an app will be:
- version 1.0.0, bundle_version 1
- version 1.0.0, bundle_version 2
- version 1.0.0, bundle_version 3
- version 1.1.0, bundle_version 1

So we have to pass both informations to be able to identify crashes.

## Alternative solution
One alternative solution is to change the version to be something like `1.0.0:ota1` for version 1.0.0, bundle_version 1.
That way we don't have to add a new field, and everything that works for version still works here, so that looks like a good idea to me.

The current limitation we have is that it is not currently possible to set the version in the native SDKs.

Let me check with the rest of the mobile team if that is something that we can do.

Reply:
IMO it's better to leverage existing `version` and simply embed OTA version in it, like you propose in alternative solution @louiszawadzki. I'm only not sure if colon is safe, but dash definitely is (it's used in semver):
```
version = "<native app version>-ota<js bundle version>"
```
This way, every `version` will be uniquely determining the version of RN codebase (both the native and OTA code).

What I dislike about introducing `bundle_version` is that it leads to ambiguity. Both `version:1.0.0` and `bundle_version:3` queries will return results coming from more than one version of RN software. Given that `version` is very important to Datadog's [Unified Service Tagging](https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging) I'm not sure if we want to allow it.

Reply:
I am actually on the same page with @ncreated on this one just for more visibility. You could also work with the `version` and `bundle_version` but having both values in one single property looks cleaner to me. Also you will only have one single facet this way which will reflect both versions for an app.

Reply:
That makes sense to me.
I'm creating tasks to enable versions in SDKs.

We can still later add `bundle_version` if that is relevant later.